packages/contracts/src/Game.sol

```diff
@@ -496,30 +508,39 @@ contract Game {
     // (swap removed card with last card, and truncate the array by one).
     //
     // The player's deck is cards[pdata.deckStart:pdata.deckEnd].
-    function checkInitialHandProof(PlayerData storage pdata, bytes32 initialDeckRoot,
-             uint256 randomness, uint256 committedSalt, bytes calldata proof) view internal {
+    function checkInitialHandProof(
+        PlayerData storage pdata,
+        bytes32 initialDeckRoot,
+        uint256 randomness,
+        uint256 committedSalt,
+        bytes calldata proof
```

Comment:
With the new snarkjs version where the verifier contract takes in `uint256[24]` as proof, I'm wondering if the client side proof generation will also generate `BigInt[24]` instead of hex string? (Assuming we are using `snarkjs.plonk.fullProve(...)` on the client side?)

Reply:
I'm guessing this might be the case indeed, will try it out when I get back to integrating proofs.